src/main/java/lotto/controller/LottoController.java

```diff
@@ -6,22 +6,26 @@
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
-import java.util.List;
-
 public class LottoController {
     public static void main(String[] args) {
-        String money = InputView.money();
+        Integer money = InputView.money();
```

Comment:
```suggestion
        Money money = new Money(InputView.money());
```

> 객체지향 생활 체조 원칙 
규칙 3: 모든 원시값과 문자열을 포장한다.

Money라는 객체가 있다면, `Integer Money`에 대한 검증에 대한 책임을 가질수 있지 않을까요?